sstables_loader.hh

```diff
@@ -87,8 +87,7 @@ public:
     /**
      * Download new SSTables not currently tracked by the system from object store
      */
-    future<tasks::task_id> download_new_sstables(sstring ks_name, sstring cf_name,
-            sstring endpoint, sstring bucket, sstring snapshot);
+    future<tasks::task_id> download_new_sstables(sstring ks_name, sstring cf_name, sstring endpoint, sstring bucket, sstring prefix);
```

Comment:
I see the change in method signature, but don't see the updated callers. Does it compile at all?

Reply:
Reverted